packages/docs/src/lang/eo-UY/components/DataIterators.json

```diff
@@ -34,6 +34,7 @@
     },
     "v-data-footer": {
       "disableItemsPerPage": "crwdns41102:0crwdne41102:0",
+      "hideItemsPerPage": "translate('Hide items-per-page dropdown')",
```

Comment:
I don't know how to translate in eo-UY that's why

Reply:
you dont. you only need to add to `en`. `eo-UY` gets generated from crowdin when we upload `en` for translating

Reply:
ok, thanks for the update